src/main/resources/public/index.html

```diff
@@ -3,7 +3,42 @@
 <head>
     <title>Quota-Billing</title>
 </head>
-
 <body>
+<div>
+    <h1>Log-In</h1>
+    <input type="text" id="logInName" placeholder="Username">
+    <input type="password" id="logInPassword" placeholder="Password">
+    <button type="button" id="logInButton">Log-In</button>
+</div>
+<div>
+    <h1>Sign-Up</h1>
+    <input type="text" id="signUpName" placeholder="Username">
+    <input type="password" id="signUpPassword" placeholder="Password">
+    <button type="button" id="signUpButton">Sign-Up</button>
+</div>
+
+<script src="./jquery.min.js"></script>
+<script>
```

Comment:
Should probably move this to another file... can do that as a refactor if we add more functionality.